server/src/main/java/io/crate/execution/dml/RawIndexer.java

```diff
@@ -88,7 +88,7 @@ public List<Reference> collectSchemaUpdates(IndexItem item) throws IOException {
             List<Reference> targetRefs = new ArrayList<>();
             for (String key : keys) {
                 ColumnIdent column = new ColumnIdent(key);
-                Reference reference = table.getReference(column);
+                Reference reference = getRef.apply(column);
```

Comment:
Minor: getRef is used in the first [commit](https://github.com/crate/crate/pull/14726/commits/68d0ec576a35d29cb1e61711349d54b37b1bfdec)
but actually declared in the second [commit](https://github.com/crate/crate/pull/14726/commits/aaf56d31409f640d896711897ca26d435ec8478f).

Probably leftovers from the dropped commit